ci-operator/step-registry/install-operators/install-operators-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: install-operators
+  from: cli-jq
```

Comment:
I feel if this is going to be a step that is global to everyone (available in the step registry), then as an end user. All they should need to do is include this new step within their job config (e.g. `- ref: install-operators`) along with an image already available in the cluster. Any additional steps such as creating the container image to use for the step should be avoided.

As taking ownership of this new step, it would make sense to contribute a new base image available to all OCP clusters within prow. This way the user can just define the base image over needing to do a `dockerfile_literal` within every job config using this step.

Based on my knowledge, we would need to do the following to have a new base image be made available to the clusters:

1. Create a new repository for this step with the Dockerfile for building the image (probably repo in the CSPI-QE org)
2. Wire up a job to automatically build the image and push the image to a container image registry for every PR merge
3. Open PR to release repository to include this new image to be mirrored into the clusters making it available for use.

Example how `osde2e` mirrors their image into the clusters: https://github.com/openshift/release/blob/master/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci#L27

https://docs.ci.openshift.org/docs/how-tos/external-images/#mirror-public-images

Then from the job config they just need to declare it within the base images:

```
base_images:
  osde2e:
    name: osde2e
    namespace: ci
    tag: latest
```

So we could have a base image for this step such as:

```
base_images:
  install-operators:
    name: install-operators
    namespace: ci
    tag: latest
```

Reply:
I like that, I wasn't aware we had any control over which images get put into the image registry. 

@ascerra are you okay with this approach too?

Reply:
@calebevans Since this is quite a simple container image, the repo created could even benefit from using the [GH container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry). I've used it before, works nicely and can tie it into a GH action.

Reply:
@ryankwilliams yeah I can give that a shot. Do you think this should be its own repo though? Maybe we can just do a lp-interop-images or cspi-images repo for this sort of thing?